docs/internal_contributors.md

```diff
@@ -0,0 +1,70 @@
+# Internal i.AI contributors to ThemeFinder
+
+At present we are not accepting contributions to `themefinder` as it is undergoing rapid development within i.AI. If you have a suggestion, please raise an [issue](https://github.com/i-dot-ai/themefinder/issues), fill in our [feedback form](https://forms.gle/85xUSMvxGzSSKQ499), or email us at [packages@cabinetoffice.gov.uk](mailto:packages@cabinetoffice.gov.uk).
+
+
+## Developing the package locally
+
+Ensure you have installed `pre-commit` in the repo: `pre-commit install`. This includes hooks to prevent you committing sensitive data. This will not catch everything; always take care when working in the open.
+
+We use `poetry` to develop the `themefinder` - `poetry install` to install packages.
+
+If you wish to install your local development version of the package into another project to test it, you will need to install in "editable" mode:
+```
+pip install -e <FILE_PATH>
+```
+or 
+```
+poetry add -e <FILE_PATH>
+```
+where `<FILE_PATH>` is the location of your local version of `themefinder`.
+
+
+## Evaluation
+
+The `evals/` directory contains our benchmarking evaluation suite used to measure system performance. When you make a change to the `themefinder` pipeline, run the evaluations to ensure you haven't reduced performance. 
+
+The `eval_mapping` and `eval_sentiment` evaluations use sensitive data stored in our AWS environment. These specific evaluations will only function with proper AWS account access and credentials. Similarly, the `make run_evals` command assumes you have AWS access configured.
+
+These evaluations use the Azure Open AI endpoint.
+
+### Running the evaluations
+
+Set your environment variables: copy `.env.example` to `.env` and populate with the name of the S3 bucket, and the details for the Azure endpoint.
+
+Install packages for this repo: `poetry install`.
+
+Ensure you have AWS access set-up, and assume your AWS role to allow you to access the data.
```

Comment:
think this is just 'have AWS access set up' without the hyphen

Reply:
Now updated https://github.com/i-dot-ai/themefinder/pull/17/commits/2bb731a8058dd7067340e994ef50f2259f4a896f